models/articles.models.js

```diff
@@ -11,10 +11,23 @@ exports.fetchArticleById = (article_id) => {
     });
 };
 
-exports.fetchArticleData = (sort_by = "created_at") => {
+exports.fetchArticleData = (sort_by = "created_at", topic) => {
   let query = `SELECT articles.author, articles.title, articles.article_id, articles.topic, articles.created_at, articles.votes, articles.article_img_url,  COUNT(comments.comment_id) AS comment_count FROM articles LEFT JOIN comments ON articles.article_id = comments.article_id GROUP BY articles.article_id`;
 
   const allowedQuerySort = ["created_at"];
+  const allowedTopics = ["mitch", "cats"];
```

Comment:
rather than greenlisting these topics because these are only relevant to the current data it would be better to make a function that checked if a topic is in the database